WordPress/Classes/Networking/ReaderTopicServiceRemote.m

```diff
@@ -174,9 +168,24 @@ - (NSString *)sanitizeTopicNameForAPI:(NSString *)topicName
         topicName = [topicName stringByReplacingOccurrencesOfString:@"--" withString:@"-"];
     }
 
+    topicName = [topicName stringByAddingPercentEscapesUsingEncoding:NSUTF8StringEncoding];
+
     return topicName;
 }
 
+- (NSArray *)normalizeMenuTopicsList:(NSArray *)rawTopics subscribed:(BOOL)subscribed recommended:(BOOL)recommended
+{
+    NSMutableArray *topics = [NSMutableArray array];
+    for (NSDictionary *topicDict in rawTopics) {
+        // Failsafe
+        if (![topicDict isKindOfClass:[NSDictionary class]]) {
```

Comment:
I wonder if it'd be better to check for this within `normalizeTopicDictionary` so it's future proof. I am not sure about this, so feel free to ignore me :)